Fixes to the S3 walk-through (#1231)
datalad-handbook/book: docs/basics/101-139-s3.rst

   A newly created public S3 bucket

By default, this bucket and its contents are not publicly accessible.
To make them public, switch to the "Permissions" tab in your buckets S3 console overview, and turn the option "Block all public access" off.

.. figure:: ../artwork/src/aws_s3_bucket_permissions.png

   Bucket settings allow making the bucket public

.. find-out-more:: Info on public buckets created prior to April 2023

>     Amazon S3 buckets created after April 2023 had support for using ACLs for public read access to files.

Reviewer: ```suggestion
    Amazon S3 buckets created before April 2023 supported using ACLs for public read access to files.
```
